src/hotspot/share/utilities/decoder_elf.cpp

```diff
@@ -67,7 +67,7 @@ bool ElfDecoder::get_source_info(address pc, char* filename, size_t filename_len
   char filepath[JVM_MAXPATHLEN];
   filepath[JVM_MAXPATHLEN - 1] = '\0';
   int offset_in_library = -1;
-  if (!os::dll_address_to_library_name(pc, filepath, sizeof(filepath), &offset_in_library)) {
+  if (!os::dll_address_to_library_name(pc, filepath, (int)sizeof(filepath), &offset_in_library)) {
```

Comment:
How about fixing dll_address_to_library_name to use size_t like all the rest of the os:: functions?

Reply:
It's painful. See above.

Reply:
I didn't need that cast.  reverting.